bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -1151,9 +1205,9 @@ DependsOnLatAndVeg - Arctic vegetation depends on latitude as above, but tempera
 </entry>
 
 <entry id="use_snicar_frc" type="logical" category="physics"
-       group="clm_inparm" valid_values="" value=".false.">
+       group="snicar_inparm" value=".true.">
 Toggle to turn on calculation of SNow and Ice Aerosol Radiation model (SNICAR) radiative forcing
-(snicar_frc=".true." is EXPERIMENTAL NOT SUPPORTED!)
+<default>Default: .true.</default>
```

Comment:
> Is this "use_snicar_frc" true or false for phys="clm5_0"? I did not realize it is false in clm5_0, but this probably does not matter since it should only affect some diagnostic variable calculations and output.

I assumed it was .false. based on the above lines. Now I see that the description of this namelist variable may be wrong. It really just determines whether snicar variables go to restart and history. 

- Would you rather keep it .true. always?
- Could you recommend a more appropriate name and more appropriate description for it?

Reply:
If .true. always, then maybe we can remove it altogether?

Reply:
I think we can always keep it true and remove it altogether. This will only add a few more variables to the restart, and would probably not increase the restart file size too much, compared to setting it to false.

Reply:
Ok, though might you recommend one of the other switches to shorten the list of restart variables? If not, that's fine. I just thought I'd check.

Reply:
I do not have any specific recommendation for this. Anything works best for CLM based on your opinion should be fine. If you would prefer keep it false as default, it is OK with me. We could rename this namelist control from "use_snicar_frc" to something like "snicar_aer_forcing" or "snicar_aerforc_diag" (physical meaning: calculate the snow albedo forcing diagnostics for each aerosol species in CLM-SNICAR).

Reply:
Renaming and setting to .false. as default sounds better to me if it's only activating diagnostics. Also, I still wonder whether these diagnostics need to be on restart files? The reason would be if they change answers by being on restart, in which case they are not just diagnostics anymore.

Reply:
...with this, too, I see that your goal was to not change the existing operation of the code, so I will not second-guess and leave as you have it, but making it default .false.

Reply:
sounds good to me. Thank you!